sensor/common/registry/registry_store.go

```diff
@@ -26,6 +27,9 @@ type Store struct {
 	// store maps a namespace to the names of registries accessible from within the namespace.
 	store map[string]registries.Set
 
+	// globalRegistries holds registries global to openshift (applies to multiple namespaces)
```

Comment:
I don't fully follow the comment. Only OpenShift? Is this the "global pull secret" stuff?

also nit: OpenShift

Reply:
Correct, this is the global pull secret stuff that can apply to any namespace in OpenShift - and is utilized regardless of namespace, specified pull secrets, or service accounts.

Reply:
I updated the comment to be more descriptive